front-end/e-learning/src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Login from './Login';
+
+class App extends React.Component {
+
+  render() {
+    return (
+      <div>
+          <Login />
+      </div>
+    );
+  }
+}
+
+export default App;
```

Comment:
add new line at the end of every file

```suggestion
export default App;

```